<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddBaseInfoDialog</class>
 <widget class="QDialog" name="AddBaseInfoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>270</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>360</width>
    <height>270</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>360</width>
    <height>270</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>添加基础数据信息</string>
  </property>
  <widget class="QGroupBox" name="box_static">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>341</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string>Static</string>
   </property>
   <widget class="QLabel" name="label_static">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>41</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>Static</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_description">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>75</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>描述：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_static">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>36</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>50</number>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="edit_description">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>70</y>
      <width>191</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_ok">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>230</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cancel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>230</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_cancel</sender>
   <signal>clicked()</signal>
   <receiver>AddBaseInfoDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>241</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
